src/cluster-legacy.c

```diff
@@ -3129,7 +3019,7 @@ int clusterProcessPacket(clusterLink *link) {
         /* If our config epoch collides with the sender's try to fix
          * the problem. */
         if (sender &&
-            nodeIsMaster(myself) && nodeIsMaster(sender) &&
+                clusterNodeIsMaster(myself) && clusterNodeIsMaster(sender) &&
```

Comment:
nit: Spacing is weird here and not aligned.
```suggestion
            clusterNodeIsMaster(myself) && clusterNodeIsMaster(sender) &&
```

Reply:
fixed